src/components/Fields/MarkdownLabel.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactMarkdown from 'react-markdown';
+import PropTypes from 'prop-types';
+import { ALLOWED_MARKDOWN_LABEL_TAGS } from '../../utils/config';
+
+const MarkdownLabel = ({ label }) => (
+  <ReactMarkdown
+    source={label}
+    allowedTypes={ALLOWED_MARKDOWN_LABEL_TAGS}
+  />
+);
+
+MarkdownLabel.propTypes = {
+  label: PropTypes.string,
```

Comment:
I'd like this to be a required prop. On a conceptual level if there's no label we don't need to render this component. We should handle conditional rendering at the level of the component above this IMO.

Reply:
Done.